app/controllers/system/admin/instance_announcements_controller.rb

```diff
@@ -0,0 +1,46 @@
+class System::Admin::InstanceAnnouncementsController < System::Admin::Controller
+  load_and_authorize_resource :announcement, through: :current_tenant, parent: false,
+                              class: Instance::Announcement.name
+  add_breadcrumb :index, :admin_instance_announcements_path
+
+  def index
+    @announcements = @announcements.includes(:creator).page(params[:page])
+  end
+
+  def new
+  end
+
+  def create
+    if @announcement.save
+      redirect_to admin_instance_announcements_path,
+                  success: t('.success', title: @announcement.title)
+    else
+      render 'new'
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @announcement.update_attributes(announcement_params)
+      redirect_to admin_instance_announcements_path,
+                  success: t('.success', title: @announcement.title)
+    else
+      render 'edit'
+    end
+  end
+
+  def destroy
+    if @announcement.destroy
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.